task_processing/plugins/mesos/logging_executor.py

```diff
@@ -57,6 +68,15 @@ def __init__(self, downstream_executor):
         self.logging_thread.daemon = True
         self.logging_thread.start()
 
+    def log_line(self, stream, line, task_id, container_id, agent):
+        formatted_line = self.format_string.format(
```

Comment:
not a blocker, but maybe we don't want to pre-format the message? i'd leave that to the user.

Reply:
The user doesn't have the container_id or agent info, unless you want to pass those to the handler, too?